app/src/components/FileUpload/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { StyledFileUpload, Label } from './styledComponents';
+
+export const FileUpload = () => (
+  <Label for="file-upload" class="custom-file-upload">
+    <StyledFileUpload id="file-upload" type="file" inputProps={{ accept: 'image/*' }} />
```

Comment:
I believe `id` is something that should be unique, so wouldn't it be better if a programmer could pass it as argument?
```suggestion
  <Label for={fileUploadId} class="custom-file-upload">
    <StyledFileUpload id={fileUploadId} type="file" inputProps={{ accept: 'image/*' }} />
```